app/config.py

```diff
@@ -53,7 +53,13 @@ class Config(object):
     PERMANENT_SESSION_LIFETIME = 1800  # 30 Minutes
     SEND_FILE_MAX_AGE_DEFAULT = 365 * 24 * 60 * 60  # 1 year
     REPLY_TO_EMAIL_ADDRESS_VALIDATION_TIMEOUT = 45
-    ACTIVITY_STATS_LIMIT_DAYS = 7
+    ACTIVITY_STATS_LIMIT_DAYS = {
+        "today": 0,
+        "one_day": 1,
+        "three_day": 3,
+        "five_day": 5,
+        "seven_day": 7,
+    }
```

Comment:
This could be an enum (and put in `enum.py`):

```
class ActivityStatsLimitDays(enum.IntEnum):
    TODAY: 0
    ONE_DAY: 1
    THREE_DAY: 3
    FIVE_DAY: 5
    SEVEN_DAY: 7
```

Then it can be built out into the DB as necessary in models.py (see how other enums are being used). This makes things cleaner with the DB. I might recommend also changing the name to something like ACTIVITY_STATS_LIMIT since "DAYS" becomes redundant with the names of the enums.